<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QPushButton{
color:#fff;
	
	font: 18pt &quot;Arial Black&quot;;
	background-color: rgb(167, 144, 28);
border-radius:5px;





  border: 2px solid #000;
  padding: 10px 20px;

}

@keyframes pulse {
  0% {
    transform: scale(1);
  }
  50% {
    transform: scale(1.2);
  }
  100% {
    transform: scale(1);
  }


}
QLabel{
font: 87 48pt &quot;Arial Black&quot;;
text-decoration: underline;
color: rgb(162, 139, 46);
text-align: center;}

.QMainWindow{

	background-color: rgb(255, 255, 255);

}
.QLineEdit{

border-radius:5px;
font: 87 16pt &quot;Arial Black&quot;;
color: rgb(152, 130, 19);
 border-style: solid ;
   border-color: rgb(178, 132, 16);
   border-width: 5px;
}




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">margin:5px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TP - LIA Mohadi Amar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_screen">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777206</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">fontsize:24pt;</string>
        </property>
        <property name="placeholderText">
         <string>Your Expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Postfix_screen">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777206</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string>Postfix </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="compile">
          <property name="text">
           <string>compile</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="validat">
          <property name="text">
           <string>Validation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="drow">
          <property name="text">
           <string>Drow</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="dell">
          <property name="text">
           <string>Dell</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="clear">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_13">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>k</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_11">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Impact&quot;;</string>
          </property>
          <property name="text">
           <string>↔</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_10">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>q</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_19">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>p</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_16">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>s</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButton_15">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_17">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Impact&quot;;</string>
          </property>
          <property name="text">
           <string>¬</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_6">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_20">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Impact&quot;;</string>
          </property>
          <property name="text">
           <string>∪</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_14">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Impact&quot;;</string>
          </property>
          <property name="text">
           <string>→</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>c</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>b</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_18">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_5">
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>e</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_1">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Impact&quot;;</string>
          </property>
          <property name="text">
           <string>∩</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
